modules/create-a-containerruntimeconfig-crd.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+[id="create-a-containerruntimeconfig_{context}"]
+
+= Creating a ContainerRuntime CR to edit CRI-O parameters
+The ContainerRuntimeConfigs custom resource definition (CRD) provides a
+structured way of changing settings associated with the {product-title}
+CRI-O runtime. Using a ContainerRuntimeConfigs custom resource (CR),
+you select the configuration values you want and the MCO handles rebuilding
+the `crio.conf` and `storage.conf` configuration files.
+
+Parameters you can set in a ContainerRuntimeConfig CR include:
+
+* **PIDs limit**: Sets the maximum number of processes allowed in a container.
+By default, the limit is set to 1024 (`pids_limit = 1024`).
+* **Log level**: Sets the level of verbosity for log messages. The default is
+`info` (`log_level = info`). Other options include `fatal`, `panic`, `error`,
+`warn`, `debug`, and `trace`.
+* **Overlay size**: Sets the maxim size of a container image. The default is
+10GB.
+* **Maximum log size**: Sets the maximum size allowed for the container
+log file. The default maximum log size is unlimited (`log_size_max = -1`).
+If it is set to a positive number, it must be at least 8192
+to not be smaller than conmon's read buffer.
+
+The following procedure illustrates how to change CRI-O settings using the
+ContainerRuntimeConfig CR.
+
+.Procedure
+
+. To raise the pidsLimit to 2048, set the logLevel to debug, and
+set the overlaySize to 8GB, create a custom resource file
+(for example, `overlay-size.yaml`) that contains that setting:
++
+[source,yaml]
+----
+$ cat << EOF > /tmp/overlay-size.yaml
+apiVersion: machineconfiguration.openshift.io/v1
+kind: ContainerRuntimeConfig
+metadata:
+ name: overlay-size
+spec:
+ machineConfigPoolSelector:
+   matchLabels:
+     custom-crio: overlay-size
+ containerRuntimeConfig:
+   pidsLimit: 2048
+   logLevel: debug
+   overlaySize: 8G
+EOF
+----
+
+. To apply the ContainerRuntimeConfig settings, run:
++
+[source,terminal]
+----
+$ oc create -f /tmp/overlay-size
+----
+
+.. Run:
```

Comment:
This threw me off...a nested "run" task inside another. Might help to tell the user the goal or purpose here, such as, "To verify that the settings were applied, run:" (or whatever it is they're using this for).